pyfar/classes/audio.py

```diff
@@ -393,7 +389,7 @@ class FrequencyData(_Audio):
     incomplete spectra.
 
     """
-    def __init__(self, data, frequencies, fft_norm=None, comment=None,
+    def __init__(self, data, frequencies, fft_norm='none', comment=None,
```

Comment:
I'd suggest the remove the obsolete parameter and move the corresponding explanation in the docstring as `Notes`
```suggestion
    def __init__(self, data, frequencies, comment=None,
```